mediathread/main/views.py

```diff
@@ -804,15 +804,23 @@ def send_faculty_email(form, faculty_user):
         subject = 'Your Mediathread Course Activation: {}'.format(
             data.get('course_name'))
         body = """
-Course Name: {}
-Term: {}
-Year: {}
-Consult or Demo: {}
-""".format(data.get('course_name'),
-           data.get('term'),
-           data.get('year'),
-           data.get('consult_or_demo'))
-
+Dear {},
+
+Thank you for creating your Mediathread course {}
+https://mediathread.ccnmtl.columbia.edu/
```

Comment:
I just copied the format that was in the google document. I guess it doesn't really make sense to have it here, I will remove it. I think Mark was implying that we want to include a link to the mediathread course, courses don't have addressable URLs yet.